tests/test_plugin.py

```diff
@@ -90,3 +90,30 @@ def test_account(fxa_account):
     """)
     result = testdir.runpytest()
     result.assert_outcomes(passed=1)
+
+
+def test_fxa_env_env_variable(monkeypatch, testdir):
+    env = 'stable'
+    monkeypatch.setenv('FXA_ENV', env)
+    testdir.makepyfile("""
+        import pytest
+
+        def test_account(fxa_urls):
+            assert '{0}' in fxa_urls['authentication']
+    """.format(env))
+    result = testdir.runpytest()
+    result.assert_outcomes(passed=1)
+
+
+def test_fxa_env_marker(monkeypatch, testdir):
+    env, marker = ['stable', 'stage']
```

Comment:
It would be good to have an extra test that makes sure the marker can handle a sequence of environments.

Reply:
Agreed, coming up!

Reply:
Great! 🚀